src/lib.rs

```diff
@@ -113,6 +113,7 @@ pub async fn init() -> Result<()> {
     check_dependencies()?;
 
     // Navigate to the program folder and run `cargo build-sbf`
+    // TODO: do we really need this?
```

Comment:
this checks for the `program` folder in base of the project, not inside `src/app/program`
is it the intended behaviour?